roles/docker_metadata/tasks/pyff.yml

```diff
@@ -16,15 +17,14 @@
     dest: "{{ metadata_basedir }}/certs/{{ item.file }}"
     content: "{{item.contents}}"
     mode: "{{item.mode}}"
-    owner: "root"
+    owner: "{{metadata_user}}"
     group: "{{metadata_group}}"
```

Comment:
Dit lijkt me juist niet de bedoeling; idee is dat de metadata user juist geen schrijfrechten heeft op die files.

Reply:
Anders wil pyff niet starten